api/src/entities/user/user.service.ts

```diff
@@ -49,14 +50,14 @@ export class UserService {
     }
 
     softDelete = async (user: UserInterface) => {
-        const deleted = await User.update({ isDeleted: true }, {
-            where: {
-                id: user.id
-            }
+        Connection.transaction(async transaction => {
+            await User.update({ isDeleted: true }, { where: { id: user.id }, transaction });
+            await UserGroup.destroy({ where: { userId: user.id }, transaction });
```

Comment:
According to task 4.2 (point 2):
> If any record gets removed from the DB, then all linked records should be removed from UserGroup as well.

So why do you use 'destroy' method and delete from database dependencies in  this softDelete method that doesn't delete anything from database?

Reply:
I just read this task incorrectly. Removed this from soft delete method